build.cake

```diff
@@ -216,10 +243,9 @@ Task("PackageReleaseZipFile")
         EnsureDirectoryExists(stagingDirectory);
         CleanDirectory(stagingDirectory);
 
-        var stagingNet45Directory = stagingDirectory.Combine("net45");
-        CopyDirectory(artifactsBinNet45Directory, stagingNet45Directory);
-        DeleteFile(stagingNet45Directory.CombineWithFilePath("DnsClient.dll"));
-        DeleteFile(stagingNet45Directory.CombineWithFilePath("DnsClient.xml"));
+        var stagingNet452Directory = stagingDirectory.Combine("net452");
+        CopyDirectory(artifactsBinNet452Directory, stagingNet452Directory);
+        DeleteFiles($"{stagingNet452Directory}/DnsClient.*");
```

Comment:
Cake weirdness: DeleteFile throws an exception if the file does not exist, but DeleteFiles with a pattern only emits a warning if no files match the pattern.

I don't know why there apparently used to be a DnsClient.xml file and there no longer is.